spec.emu

```diff
@@ -38,7 +89,7 @@ contributors:
       1. Perform ? RequireInternalSlot(_M_, [[WeakMapData]]).
       1. Let _entries_ be the List that is _M_.[[WeakMapData]].
       1. <del>If Type(_key_) is not Object, return *false*.</del>
-      1. <ins>If IsObjectOrSymbol(_key_) is *false*, return *false*.</ins>
+      1. <ins>If ! %CanBeHeldWeakly%(_key_) is *false*, return *false*.</ins>
```

Comment:
Do we think this is OK to direct use the intrinsic, or should I split it up into a core AO, and then the separate intrinsic which is just a wrapper around the AO?

Reply:
if you're directly using it, you'd have to `Call()` it, since it's a JS function object. it's probably better to have a common AO that both reference.

Reply:
Thanks @ljharb, I should have gone with my gut! I've updated so there is a common AO to use.